docs/docs/volumes/cache.md

```diff
@@ -13,7 +13,13 @@ Before use, please ensure that Dragonfly has been installed in the cluster and r
 
 ## Install Dragonfly
 1. Configure /etc/hosts according to the cluster
-2. Configure the default sc according to the selection
+```console
+$ vi /etc/hosts
+host1-IP hostName1
+host2-IP hostName2
+host3-IP hostName3
+```
```

Comment:
```suggestion
   ```console
   $ vi /etc/hosts
   host1-IP hostName1
   host2-IP hostName2
   host3-IP hostName3
   ```
```
Normally, indentations (3 spaces) are required for 1-2-3 list